lavamoat/browserify/beta/policy.json

```diff
@@ -1920,6 +1920,7 @@
         "@metamask/notification-services-controller>@contentful/rich-text-html-renderer": true,
         "@metamask/base-controller": true,
         "@metamask/controller-utils": true,
+        "@metamask/keyring-controller": true,
```

Comment:
Dep was added since we have introduced a runtime dependency here (importing an enum):
https://github.com/MetaMask/core/blob/b5195285c3c6892f7b0cf4428abd7d66753fb0fd/packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts#L18